example/Assets/Standard Assets/Bugsnag/Bugsnag.cs

```diff
@@ -529,7 +521,8 @@ public enum Severity {
                 errorClass = match.Groups["errorClass"].Value;
                 errorMessage = match.Groups["message"].Value.Trim();
             } else {
-                errorClass = logString;
+                errorClass = "UnityLog" + type;
```

Comment:
This is for an unrelated change? This seems to already be on master. What is going on here?